sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_feature_set/feature_set_schema.py

```diff
@@ -16,9 +16,14 @@
 class FeatureSetSchema(YamlFileSchema):
     name = fields.Str(required=True, allow_none=False)
     version = fields.Str(required=True, allow_none=False)
+    latest_version = fields.Str(dump_only=True)
```

Comment:
Should the SDK entity classes be updated as well?

Reply:
The entites already have latest_version property, the latest version will be assigned when getting from service:
https://github.com/Azure/azure-sdk-for-python/blob/1f9192a3bc3880edf9b44cc23e1cd3914b8b9b0b/sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_store_entity/feature_store_entity.py#L76
https://github.com/Azure/azure-sdk-for-python/blob/1f9192a3bc3880edf9b44cc23e1cd3914b8b9b0b/sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/_artifacts/feature_set.py#L87